addons/website/views/snippets/s_company_team_shapes.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_shapes" name="Team Shapes">
+    <section class="s_company_team_shapes o_cc o_cc2 pt48 pb48">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col-lg-12 pb24">
+                    <h2 style="text-align: center;">Our talented crew</h2>
+                </div>
+                <div class="col-6 col-lg-4 pb24">
+                    <img src="/web_editor/image_shape/website.s_company_team_image_1/web_editor/geometric_round/geo_round_square.svg" class="img img-fluid" alt="" data-shape="web_editor/geometric_round/geo_round_square" data-original-mimetype="image/jpeg" data-file-name="s_company_team_image_1.svg" data-shape-colors=";;;;" style="width: 100% !important;"/>
+                    <h3 class="h5-fs" style="text-align: center;"><br/>Tony Fred</h3>
```

Comment:
You can wrap the image in a `<p>` to get it's margin do the spacing you want, without having to add a `<br/>` 👍 Same for other occurences